standard/expressions.md

```diff
@@ -3688,6 +3817,55 @@ At run-time, the expression `await t` is evaluated as follows:
 
 An awaiter’s implementation of the interface methods `INotifyCompletion.OnCompleted` and `ICriticalNotifyCompletion.UnsafeOnCompleted` should cause the delegate `r` to be invoked at most once. Otherwise, the behavior of the enclosing async function is undefined.
 
+## §range-operator Range operator
+
+This operator provides a succinct syntax for specifying a (possibly empty) element range suitable for use in denoting a slice of an indexable sequence (§indexable-sequence).
+
+```ANTLR
+range_expression
+    : unary_expression
+    | range_expression? '..' range_expression?
```

Comment:
Is this definitely correct? That suggests `0 .. 4 .. 5` would be valid, because `0` is a valid range expression, and so is `4 .. 5`.

I think I'm missing something here.

Reply:
I haven’t yet taken a look at the grammar, but yes that looks wrong.